sda-download/api/sda/sda_test.go

```diff
@@ -777,6 +847,26 @@ func TestDownload_Whole_Range_Encrypted(t *testing.T) {
 	assert.Equal(t, 200, response.StatusCode, "Unexpected status code from download")
 	assert.Equal(t, []byte("crypt4gh"), body[:8], "Unexpected body from download")
 
+	// Test encrypted download, should work even when AllowedUnencryptedDownload is false
```

Comment:
I don't really see why this is part of the `TestDownload_Whole_Range_Encrypted` function?

Reply:
@MalinAhlberg I agree it's not the best place to put it here. But for the first, I wanted to avoid duplicating the code that prepare the data, and for the second this config setting will anyway be refactored later on. Do you think it will be better that I just remove this test?

Reply:
I see. I would vote for keeping it as is, and plan for a general refactoring of the tests later. So feel free to resolve this conversation.